lib/nosedrum/invoker/split.ex

```diff
@@ -25,6 +25,38 @@ defmodule Nosedrum.Invoker.Split do
   alias Nosedrum.{Helpers, Predicates}
   alias Nostrum.Struct.Message
 
+  @spec remove_prefix(String.t()) :: String.t() | :not_found
+  if is_list(@prefix) do
+    defp remove_prefix(message) do
+      with(
+        # Find the prefix that was used in the message, going through the list of prefixes
+        real_prefix when real_prefix != :not_found <-
+          Enum.find(
+            if(is_list(@prefix), do: @prefix, else: [@prefix]),
```

Comment:
I think we can drop this, since we now determine this at compile time, right?

```suggestion
            @prefix,
```